operate/constants.py

```diff
@@ -31,12 +37,32 @@
 KEYS_JSON = "keys.json"
 DOCKER_COMPOSE_YAML = "docker-compose.yaml"
 SERVICE_YAML = "service.yaml"
+STAKED_BONDING_TOKEN = "OLAS"
+ZERO_ADDRESS = "0x0000000000000000000000000000000000000000"
 
 ON_CHAIN_INTERACT_TIMEOUT = 120.0
 ON_CHAIN_INTERACT_RETRIES = 40
 ON_CHAIN_INTERACT_SLEEP = 3.0
 
 HEALTH_CHECK_URL = "http://127.0.0.1:8716/healthcheck"  # possible DNS issues on windows so use IP address
-
+SAFE_WEBAPP_URL = "https://app.safe.global/home?safe=gno:"
 TM_CONTROL_URL = "http://localhost:8080"
+IPFS_ADDRESS = "https://gateway.autonolas.tech/ipfs/f01701220{hash}"
 ZERO_ADDRESS = "0x0000000000000000000000000000000000000000"
+
+WRAPPED_NATIVE_ASSET = {
+    Chain.ETHEREUM.value: "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
+    Chain.POLYGON.value: "0x0d500B1d8E8eF31E21C99d1Db9A6444d3ADf1270",
+    Chain.GNOSIS.value: "0xe91D153E0b41518A2Ce8Dd3D7944Fa863463a97d",
+    Chain.ARBITRUM_ONE.value: "0x82aF49447D8a07e3bd95BD0d56f35241523fBab1",
+    Chain.OPTIMISTIC.value: "0x4200000000000000000000000000000000000006",
+    Chain.MODE.value: "0x4200000000000000000000000000000000000006",
+    Chain.BASE.value: "0x4200000000000000000000000000000000000006",
+}
+
+# TODO: These links may break in the future, use a more robust approach
+MECH_CONTRACT_JSON_URL = "https://raw.githubusercontent.com/valory-xyz/mech/refs/tags/v0.8.0/packages/valory/contracts/agent_mech/build/AgentMech.json"
+STAKING_TOKEN_INSTANCE_ABI_PATH = 'https://raw.githubusercontent.com/valory-xyz/trader/refs/tags/v0.23.0/packages/valory/contracts/staking_token/build/StakingToken.json'
+MECH_ACTIVITY_CHECKER_JSON_URL = 'https://raw.githubusercontent.com/valory-xyz/autonolas-staking-programmes/refs/heads/main/abis/0.8.25/SingleMechActivityChecker.json'
+SERVICE_REGISTRY_TOKEN_UTILITY_JSON_URL = "https://raw.githubusercontent.com/valory-xyz/open-autonomy/refs/tags/v0.18.4/packages/valory/contracts/service_registry_token_utility/build/ServiceRegistryTokenUtility.json"
+MECH_AGENT_FACTORY_JSON_URL = "https://raw.githubusercontent.com/valory-xyz/ai-registry-mech/main/abis/0.8.25/AgentFactory.json"
```

Comment:
Why aren't we using the built-in framework contracts ?

Reply:
you mean from open-autonomy? They are not there.